qemu/tests/qemu_disk_img_commit.py

```diff
@@ -46,6 +47,7 @@ def run(test, params, env):
         raise error.TestError("Fail to save tmp file")
     commit_test.destroy_vm()
     commit_test.commit()
+    utils.system("sync")
```

Comment:
Hi @CongSmile 
I think you mean whether sync should be done after commit,  yes, i have tried the sync time is very short in this case.